frontend/src/routes/Applications/CreateApplication/CreateApplicationArgo.tsx

```diff
@@ -41,9 +41,9 @@ export function WizardSyncEditor() {
     const { update } = useData() // Wizard framework sets this context
     return (
         <SyncEditor
+            editorTitle={'Argo YAML'}
```

Comment:
@fxiang1 You mentioned something about certain Argo terms not being used in the UI. Should this say **Application set YAML**

Reply:
Yep doc has been pushing to move away from anything referencing ArgoCD (see https://github.com/stolostron/backlog/issues/20621) so I think we should rename it to `Application set YAML` or something similar.